src/main/java/net/snowflake/client/core/CredentialManager.java

```diff
@@ -150,7 +178,7 @@ void writeMfaToken(SFLoginInput loginInput, SFLoginOutput loginOutput) throws SF
   synchronized void writeTemporaryCredential(SFLoginInput loginInput, String cred, String credType)
       throws SFException {
     if (Strings.isNullOrEmpty(cred)) {
-      logger.debug("no %s is given.", credType);
+      logger.debug("No %s is given.", credType);
```

Comment:
let's replace %s with {} in the log message

Reply:
Done